src/global.d.ts

```diff
@@ -4,7 +4,7 @@ import Explorer from "./Explorer/Explorer";
 declare global {
   interface Window {
     authType: AuthType;
-    dataExplorer: Explorer;
+    dataExplorer: any;

```

Comment:
This needs to be `Explorer`

Reply:
I have remove all of the files related to Explorer